src/compute/src/logging/compute.rs

```diff
@@ -544,8 +564,8 @@ struct DemuxOutput<'a> {
     frontier: OutputSession<'a, FrontierDatum>,
     import_frontier: OutputSession<'a, ImportFrontierDatum>,
     frontier_delay: OutputSession<'a, FrontierDelayDatum>,
-    peek: OutputSession<'a, Peek>,
-    peek_duration: OutputSession<'a, u128>,
+    peek: OutputSession<'a, (PeekType, Peek)>,
+    peek_duration: OutputSession<'a, (PeekType, u128)>,
```

Comment:
Could you define two new datum types, `PeekDatum` and `PeekDurationDatum`, and make them wrap the fields in these tuples? Like what we do with the other outputs if there is more than one value to transport.